packages/pyright-internal/src/localization/package.mb.en.json

```diff
@@ -0,0 +1,30 @@
+{
+    "Diagnostic": {
+        "argAssignmentParam": "Argument does not match parameter type for parameter \"{paramName}\"",
+        "argAssignmentParamFunction": "Argument does not match parameter type for parameter \"{paramName}\"",
+        "argMissingForParam": "Argument missing for parameter {name}",
+        "argMissingForParams": "Arguments missing for parameters {names}",
+        "argMorePositionalExpectedCount": "Expected {expected} more positional arguments",
+        "booleanIsLowerCase": "\"{name}\" is not defined, did you mean \"{booleanName}\"?",
+        "breakOutsideLoop": "\"break\" can be used only within a loop",
+        "continueOutsideLoop": "\"continue\" can be used only within a loop",
+        "expectedCloseBrace": "Missing closing bracket \"}\"",
+        "expectedCloseBracket": "Missing closing bracket \"]\"",
+        "expectedCloseParen": "Missing closing bracket \")\"",
+        "expectedColon": "Missing colon \":\"",
+        "expectedEqualityOperator": "Expected equality operator, did you mean \"==\"?",
+        "expectedExpr": "Expected expression",
```

Comment:
Expected expression occurs where it is missing from:

- if/while statements; e.g. `if :`
- the right hand side of operands excluding assign (`=`) which is special cased

Reply:
"excluding assign (=) which is special cased" <- should that be a common error?